### DISCUSSION TITLE
Chaning region.end on region-updated event doesn't change the region

### DISCUSSION THREAD

**Discussion by logesh2496** [katspaugh/wavesurfer.js#2997] [Q&A]
In the event "region-updated", I'm changing the region.end (since there's no onResize() method), but it doesn't change the region in the waveform.

**Comment by katspaugh** [comment]
You should call `region.setOptions` for this. I’ll add this to the docs.

**Comment by logesh2496** [comment]
it worked, thanks. Would be great if it could be added to the RegionParams type as well

**Reply by katspaugh** [reply]
It's not a param, it's a method. I've exported the entire Region class [here](https://github.com/katspaugh/wavesurfer.js/pull/2999/files#diff-b38f05df385bdeeb149b8f0a1ce9f1634cdc7ed3bafb9ab1e861a380370f13bfR60) so that it's visible in the type docs.
